lib/latitudesh.js

```diff
@@ -3,16 +3,17 @@ const dotenv = require('dotenv');
 
 dotenv.config();
 
-const loadMethods = (apiKey, origin) => {
-  requests.setHttpMethods(LatitudeSh, apiKey, origin);
+const loadMethods = (apiKey, origin, version) => {
```

Comment:
In a new version, it would be nice to change the constructor to accept an object instead of four or more parameters.